code-challenges.test.js

```diff
@@ -13,7 +13,38 @@
 
 // --------------------1) Create a function that takes in a string and returns a coded message. The coded message converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.
 
+// crate a function 
+// have it assign new variables that take in value and replaces it with a new value
+//return changed string
+
 // a) Create a test with expect statements using the variables provided.
+describe("hello", () => {
+
+    const secretCodeWord1 = "Lackadaisical"
+    const secretCodeWord2 = "Gobbledygook"
+    const secretCodeWord3 = "Eccentric"
+
+ 
+    it("returns a string that says hiCreate a function that takes in a string and returns a coded message. The coded message converts 'a' to 4, 'e' to 3, 'i' to 1, and 'o' to 0.", () => {
+  
+      
+      expect(no(secretCodeWord1)).toEqual("L4ck4d41s1c4l")
+      expect(no(secretCodeWord2)).toEqual("Gobbledygook")
+      expect(no(secretCodeWord3)).toEqual("Eccentric")
+    })
+  })
```

Comment:
Your test looks great! I am noticing that you don't have any of the Jest dependencies installed meaning that you wouldn't be able to know if your tests are working.